map_it_test.go

```diff
@@ -1071,6 +1063,145 @@ func TestMap_ExecuteOnEntries(t *testing.T) {
 	})
 }
 
+func TestMap_AddInterceptor(t *testing.T) {
+	it.MapTester(t, func(t *testing.T, m *hz.Map) {
+		ctx := context.Background()
+		prefix := "My Prefix"
+		id, err := m.AddInterceptor(ctx, &MapGetInterceptor{Prefix: prefix})
+		if err != nil {
+			t.Fatal(err)
+		}
+		defer func() {
+			if _, err := m.RemoveInterceptor(ctx, id); err != nil {
+				log.Fatal(err)
+			}
+		}()
+		v, err := m.Get(ctx, "foo")
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.Equal(t, prefix, v)
+		ok, err := m.RemoveInterceptor(ctx, id)
+		if err != nil {
+			log.Fatal(err)
```

Comment:
Is there a reason that we use log.Fatal instead of t.Fatal

Reply:
It seems I've used log.Fatal instead of t.Fatal erroneously in several places, fixed them at: https://github.com/hazelcast/hazelcast-go-client/pull/648/commits/c46f0434df79f9506aa80ebbd7854bede29b4c8b